curl.out

```diff
@@ -0,0 +1,23 @@
+<executions count='1'>
+  <execution id='49278' href='https://rundeck.cozycloud.cc/api/27/execution/49278' permalink='https://rundeck.cozycloud.cc/project/Gozy/execution/show/49278' status='succeeded' project='Gozy'>
+    <user>julie</user>
+    <date-started unixtime='1550589634879'>2019-02-19T15:20:34Z</date-started>
+    <date-ended unixtime='1550589639602'>2019-02-19T15:20:39Z</date-ended>
+    <job id='8d452c26-5a39-4162-b3b8-4da5efdf0b76' averageDuration='5204' href='https://rundeck.cozycloud.cc/api/27/job/8d452c26-5a39-4162-b3b8-4da5efdf0b76' permalink='https://rundeck.cozycloud.cc/project/Gozy/job/show/8d452c26-5a39-4162-b3b8-4da5efdf0b76'>
+      <name>[cozy.rocks - Staging] Install app on instance</name>
+      <group>03 - cozy.rocks - Staging</group>
+      <project>Gozy</project>
+      <description></description>
+      <options>
+        <option name='instance' value='testcafe.cozy.rocks' />
+        <option name='source' value='https://downcloud.cozycloud.cc/upload/photos/mobile/1.15.3-e19ee9cd80ed1fcc040f06d793552467d7068a5b/photos.tar.gz' />
+        <option name='slug' value='photos' />
+      </options>
+    </job>
+    <description>cd /root/gozy-cluster-controller/ &amp;&amp; /root/.virtualenvs/cozy/bin/python ./gozy-cli install_application "${option.slug}" "${option.source}" -i "${option.instance}" -f</description>
+    <argstring>-instance testcafe.cozy.rocks -slug photos -source https://downcloud.cozycloud.cc/upload/photos/mobile/1.15.3-e19ee9cd80ed1fcc040f06d793552467d7068a5b/photos.tar.gz</argstring>
+    <successfulNodes>
+      <node name='gozy-adm-stg' />
+    </successfulNodes>
+  </execution>
+</executions>
```

Comment:
À supprimer ? On peut pas écrire sur le `stdout` et lire à partir de là ?

Reply:
oups! that was not supposed to be in the commit!

Anyway, I used stdout, and there are no more temp file!